tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -36,6 +36,7 @@ limitations under the License.
 #include <vector>
 
 #include "absl/memory/memory.h"
+#include "tensorflow/lite/c/common.h"
```

Comment:
We don't need this here, since `model.h` imported in `label_image.h` already indirectly includes it.

We need to add it to `get_top_n.h` & `get_top_n_impl.h` - see the error message posted by @gbaned .

Reply:
Thanks. I'll add it to `get_top_n_impl.h` (since `get_top_n.h` includes  `get_top_n_impl.h` already).